desktop/extensions-sdk/dev/api/dashboard-routes-navigation.md

```diff
@@ -9,7 +9,15 @@ keywords: Docker, extensions, sdk, API
 For example, navigate to a given container logs:
 
 ```typescript
-await ddClient.desktopUI.navigate.viewContainerLogs(id);
+const id = '8c7881e6a107';
```

Comment:
@benja-M-1 The `viewContainerLogs` expects the entire container id (not short),e.g:

`46b57e400d801762e9e115734bf902a2450d89669d85881058a46136520aca28`.

I think it'd be more accurate to use an entire container id in the example to avoid devs thinking a short id would be valid.